src/main/java/com/xjeffrose/chicago/db/StorageProvider.java

```diff
@@ -21,4 +24,11 @@
   void close();
 
   void open();
-}
\ No newline at end of file
+
+  void setZkClient(ZkClient zkClient);
```

Comment:
ZKClient has some wierd issue. It does not work with the constructor. ONce the zkClient has been started, then it can be used by other classes needing it. So we had to set it explicitly after initialization